src/isomorphic/classic/types/__tests__/ReactPropTypesProduction-test.js

```diff
@@ -0,0 +1,262 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails react-core
+ */
+
+'use strict';
+
+
+describe('ReactPropTypesProduction', function() {
```

Comment:
Looks like we can delete this file now and instead add a simple test against unguarded calll to the DEV file?